source/common/grpc/grpc_web_filter.cc

```diff
@@ -130,5 +169,18 @@ Http::FilterTrailersStatus GrpcWebFilter::encodeTrailers(Http::HeaderMap& traile
   return Http::FilterTrailersStatus::Continue;
 }
 
+void GrpcWebFilter::setupStatTracking(const Http::HeaderMap& headers) {
+  cluster_ = Common::resolveClusterInfo(decoder_callbacks_, cm_);
+  if (!cluster_) {
+    return;
+  }
+  do_stat_tracking_ =
+      Common::resolveServiceAndMethod(headers.Path(), &grpc_service_, &grpc_method_);
```

Comment:
Yes, it should be aborted, I leave a TODO there to return error response when request is not a valid gRPC call.